src/pages/steps/ScopeStep.tsx

```diff
@@ -147,15 +158,12 @@ const ScopeStep = <TFormData extends FormDataUnion>({
 };
 
 const scopeStepConfiguration: StepsConfiguration<FormDataUnion> = {
-  Component: ScopeStep,
+  // eslint-disable-next-line react/display-name
+  Component: (props) => <ScopeStep {...props} />,
   name: 'scope',
   title: ({ t }) => t(`create.scope.title`),
   shouldShowStep: ({ watch, offerId, formState }) => {
-    return (
-      !offerId &&
-      !watch('typeAndTheme')?.type?.id &&
-      !formState.dirtyFields.typeAndTheme
-    );
+    return !offerId;
```

Comment:
Now scope step doesn't show in edit mode

Reply:
Which is normal since you can't change the scope in edit.
I will make the disabled state in a seperate PR :)

Reply:
I understand the confusion, I've changed the title of my PR ;)